### DISCUSSION TITLE
Website : wrong path

### DISCUSSION THREAD

**Discussion by GlaDal** [opengisch/QField#3734] [Bugs]
Hi, 
The link for the community discussion plateform in the doc website doesn't work on this page :[LINK](https://docs.qfield.org/get-started/tutorials/advanced-setup-qfc/ )
It references on this page : [LINK](https://github.com/opengisch/qfieldcloud/discussions)

**Comment by suricactus** [comment]
Thanks, will be fixed very soon. See https://github.com/opengisch/QField-docs/pull/286

**Reply by suricactus** [reply]
Done! Thanks again for the report!
